assets/js/buttons-examples.js

```diff
@@ -224,6 +224,11 @@ $(document).ready(function () {
           label: "Liquid",
           classes: "liquid-btn",
         },
+        {
```

Comment:
As @dsnehasish74 said please place it in the correct alphabetical order. Also, capitalize the first letter in the label